integration/app/test-demo/src/containers/MeetingForm.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  Flex,
+  FormField,
+  Input,
+  Modal,
+  ModalBody,
+  ModalHeader,
+  PrimaryButton,
+  useMeetingManager,
+} from 'amazon-chime-sdk-component-library-react';
+import React, { ChangeEvent, FC, FormEvent, useState } from 'react';
+import { createGetAttendeeCallback, fetchMeeting } from '../utils/api';
+
+const MeetingForm: FC = () => {
+  const meetingManager = useMeetingManager();
+  const [meetingId, setMeetingId] = useState('');
+  const [name, setName] = useState('');
+  const [errorMessage, updateErrorMessage] = useState('');
+
+  const joinMeeting = async (event: FormEvent) => {
+    event.preventDefault();
+
+    const id = meetingId.trim().toLocaleLowerCase();
+    setMeetingId(id);
+    window.localStorage.setItem('meetingId', id);
```

Comment:
can we not use localStorage :D ?

Reply:
Since I deleted all meeting end related code. No need to store the `meeingId` anymore. Deleted all `localStorage` related code.